packages/amplify-e2e-tests/overrides/override-root.ts

```diff
@@ -1,4 +1,7 @@
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
```

Comment:
nit: I would prefer to see a uuid used here

Reply:
Is there a native uuid function? I’d prefer not to pull a bunch of dependencies into the e2e test app.

Reply:
Is there a native uuid function? I’d prefer not to pull a bunch of dependencies into the e2e test app.

Reply:
it's already a dependency for the e2e-test package, but we should export a wrapper function from e2e-core (we actually might already have this). It would probably be cleaner to call something like `getShortUniqueId() => string` instead of what most of the code does today